ru/browser/browser/zen-welcome.ftl

```diff
@@ -3,18 +3,24 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 welcome-dialog-welcome = 👋 Добро пожаловать в { -brand-short-name }
-welcome-dialog-welcome-subtext = Быстро. Красиво. Приватно.
+welcome-dialog-welcome-subtext = Быстрый. Красивый. Приватный.
 welcome-dialog-get-started = Начать работу
-welcome-dialog-import = 😍 Добавьте избранное { -brand-short-name }
-welcome-dialog-import-subtext = Выберите, где вы остановились в своем последнем браузере
+
+welcome-dialog-import = 😍 Перенесите избранное в { -brand-short-name }
+welcome-dialog-import-subtext = Начните с того места, где остановились в предыдущем браузере
 welcome-dialog-import-action = Импортируйте данные
-welcome-dialog-theme = Сделайте себя дома
-welcome-dialog-theme-subtext = Персонализируйте ваш опыт с вашим любимым видом!
+
+welcome-dialog-theme = Чувствуйте себя как дома
+welcome-dialog-theme-subtext = Настройте облик браузера, как вам нравится!
```

Comment:
I just thought of wouldn't it be nicer if we use a sentence for this that is less generic and more common in Russian? Something like this:

```suggestion
welcome-dialog-theme-subtext = Сделайте { -brand-short-name } по-настоящему вашим!
```

Though I should ask @mauro-balades of whether it is okay to include `{ -brand-short-name }` here.